doc/users/project/mission.rst

```diff
@@ -0,0 +1,28 @@
+Matplotlib Mission Statement
+============================
+
+The Matplotlib developer community develops, maintains, and supports Matplotlib
+and its extensions to provide plotting tools for the Scientific Python
+Ecosystem.  Our success is based on how effectively we enable our users, across
+scientific, industrial, commercial, and educational domain, to achieve their
+goals.
+
+We believe that a diverse developer community creates the best software, and we
+welcome anyone who shares our mission and values, as described in the code of
+conduct, to the community.
```

Comment:
```suggestion
conduct.
```
Can be left out. This avoids the spitting of the sentence by "as described ..." and is implied through the context of the previous sentence.